packages/x-components/src/x-modules/related-tags/components/related-tags.vue

```diff
@@ -51,7 +51,7 @@
    * This component renders a set of [`RelatedTag`](./x-components.related-tag) components by
    * default to select from after a query is performed to fine-tune search.
    * For example, if you are searching for *lego*, different related tags could be *city*,
-   * *friends*, or *harry potter*, refining the search with *lego city*, *lego friends*,
+   * friends*, or *harry potter*, refining the search with *lego city*, *lego friends*,
```

Comment:
**issue:** revert change

Reply:
Oops! the eslint did it automatically on the pre commit hook. I reverted the change but also created this https://github.com/empathyco/x/pull/1029 PR to solve the root cause.